src/components/crop-content/CropContent.tsx

```diff
@@ -10,6 +10,8 @@ type CropContentProps = {
   className?: string;
   /** If true, the crop will be collapsed initially. */
   initCollapsed?: boolean;
+  /** Control the collapsed state, otherwise it is uncontrolled */
+  collapsed?: boolean;
```

Comment:
is it possible to deprecate maybe initCollapsed?

Reply:
I think that both still make sense, initCollapsed is just at the initialisation, but collapsed you need to provide the logic to change it